react-app/webpack/webpack.dev.js

```diff
@@ -0,0 +1,7 @@
+const {merge} = require('webpack-merge');
```

Comment:
I would rather recommend having 3 parts of your `webpack.config.js`: a) common; b) development-mode relevant; c) production-mode relevant. This way you may use `merge()` conditionally just once within your main `webpack.config.js` based on which mode you're running.